--- lib/v8/v8/include/v8-internal.h.orig	2024-09-17 01:53:34.000000000 -0400
+++ lib/v8/v8/include/v8-internal.h	2024-09-17 01:53:40.000000000 -0400
@@ -1432,7 +1432,7 @@
 template <typename Iterator>
 struct MaybeDefineIteratorConcept<
     Iterator, std::enable_if_t<kHaveIteratorConcept<Iterator>>> {
-  using iterator_concept = Iterator::iterator_concept;
+  using iterator_concept = typename Iterator::iterator_concept;
 };
 // Otherwise fall back to `std::iterator_traits<Iterator>` if possible.
 template <typename Iterator>
@@ -1445,7 +1445,7 @@
   // TODO(pkasting): Add this unconditionally after dropping support for old
   // libstdc++ versions.
 #if __has_include(<ranges>)
-  using iterator_concept = std::iterator_traits<Iterator>::iterator_concept;
+  using iterator_concept = typename std::iterator_traits<Iterator>::iterator_concept;
 #endif
 };
 
